alf/examples/tutorial/ac_render_conf.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) 2021 Horizon Robotics and ALF Contributors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from functools import partial
+
+import torch
+
+import alf
+from alf.algorithms.actor_critic_algorithm import ActorCriticAlgorithm
+from alf.examples import ac_cart_pole_conf
+
+import alf.summary.render as render
+
+
+class ACRenderAlgorithm(ActorCriticAlgorithm):
+    def predict_step(self, inputs, state):
+        alg_step = super().predict_step(inputs, state)
+        action = alg_step.output
+        action_dist = alg_step.info.action_distribution
+        with alf.summary.scope("ACRender"):
```

Comment:
Nit: since we are using `'` below in this file, shall we replace `"` with `'` for slightly better conistency?